src/Components/Dashboard/Dashboard.js

```diff
@@ -8,7 +7,7 @@ import EventModal from '../EventModal/EventModal'
 
 
 const Dashboard = () => {
-  const {loading, error, data} = useQuery(GET_USER_BY_ID, {
+  const {loading, error, data, refetch} = useQuery(GET_USER_BY_ID, {
     variables: {"id": process.env.REACT_APP_USER_ID}
```

Comment:
Oh I see you're using a refetch! I assume this is related to the updated event when you rsvp ?

Reply:
Yup! I attached it to the close modal so that it triggers a refetch to populate the new rsvp'd event in the carousel